os/board/rtl8730e/src/component/mbed/targets/hal/rtl8730e/pwmout_api.c

```diff
@@ -132,7 +132,7 @@ void pwmout_init(pwmout_t *obj, PinName pin)
 
 	obj->pwm_idx = pwm_chan;
 	obj->period = 0x10000 * (prescaler + 1) / 40;
-	obj->pulse = 0x1000 * (prescaler + 1) / 40;
+	obj->pulse = (float) (0x1000 * (prescaler + 1) / 40);
```

Comment:
@ZhenBei-Sin Could you check this typecasting as well? It's the same as previous comment.

Reply:
Changed and Verified:

![image](https://github.com/Samsung/TizenRT/assets/94035727/ef6280a9-7453-4144-89fb-334bdfa8db54)